camera/macos/camera.mm

```diff
@@ -229,7 +229,6 @@ - (void)processFrame:(CMSampleBufferRef)sampleBuffer
 {
     if (m_impl->wrapped != nullptr) {
         [m_impl->wrapped stop];
-        [m_impl->wrapped release];
```

Comment:
Why was this removed?

Reply:
When I was trying to use the camera module in OEP-macos, I ran into the error: «ARC forbids explicit message send of «release»». But now I don’t use the camera module, so, if you think that this line is needed, I can return it back.

Reply:
Based on your statement ARC should be enabled for the module if this file was included in the configuration. Think about the consequences of your change if ARC is not enabled for the project.

Reply:
Update module config to enable ARC always.